tests/README.md

```
@@ -8,4 +8,8 @@ CONFIG_FILE=tests/integration/config/integration.yml cargo run
 
 ###
 Internal memory performance test:
-`cargo test --test memory_test --features integration-test  --release -- --nocapture`
\ No newline at end of file
+`cargo test --test memory_test --features integration-test  --release -- --nocapture`
+
+###
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)